.github/workflows/ci.yml

```diff
@@ -14,3 +14,37 @@ jobs:
     # Only run cron on the silverstripe account
     if: (github.event_name == 'schedule' && github.repository_owner == 'silverstripe') || (github.event_name != 'schedule')
     uses: silverstripe/gha-ci/.github/workflows/ci.yml@v1
+    with:
+      # set phpunit to false to prevent automatic generation of mysql phpunit jobs
+      phpunit: false
+      extra_jobs: |
+        - php: 8.1
+          db: sqlite3
+          phpunit: true
```

Comment:
We don't need to specify `phpunit: true` - it's true by default https://github.com/silverstripe/gha-ci/blob/1/.github/workflows/ci.yml#L38

Reply:
DONE

Reply:
@emteknetnz I think you do have to specify it if you define in as false on the level up.